onnxruntime/contrib_ops/cpu/bert/qembed_layer_norm.cc

```diff
@@ -64,22 +60,22 @@ Status ComputeInternal(OpKernelContext* context, float epsilon) {
       has_segment_embedding ? static_cast<int>(segment_embedding->Shape()[0]) : 0;
 
   // Grab quantization values:
-  float word_embedding_scale_data = *(word_embedding_scale->template Data<float>());
-  T2 word_embedding_zero_point_data = *(word_embedding_zero_point->template Data<T2>());
-
-  float position_embedding_scale_data = *(position_embedding_scale->template Data<float>());
-  T2 position_embedding_zero_point_data = *(position_embedding_zero_point->template Data<T2>());
-
-  float segment_embedding_scale_data =
-      has_segment_embedding ? *(segment_embedding_scale->template Data<float>()) : 0.0f;
-  T2 segment_embedding_zero_point_data =
-      has_segment_embedding ? *(segment_embedding_zero_point->template Data<T2>()) : 0;
-
-  float gamma_scale_data = *(gamma_scale->template Data<float>());
-  T2 gamma_zero_point_data = *(gamma_zero_point->template Data<T2>());
-
-  float beta_scale_data = *(beta_scale->template Data<float>());
-  T2 beta_zero_point_data = *(beta_zero_point->template Data<T2>());
+  quantization::Params<T2> word_embedding_params =
```

Comment:
>T2 [](http://example.com/codeflow?start=23&length=2)

nit: could we use a more meaningful type name

Reply:
Sure - done. We should uniformally make a stance here and update the style guide. I see a bunch of kernels using T1/T2/T3/T/etc. I see that even stuff in `contrib_defs.cc` also use this style.